tstyche.config.json

```diff
@@ -1,15 +1,16 @@
 {
   "testFileMatch": [
-    "**/packages/expect/__typetests__/*.test.ts",
-    "**/packages/expect-utils/__typetests__/*.test.ts",
-    "**/packages/jest/__typetests__/*.test.ts",
-    "**/packages/jest-cli/__typetests__/*.test.ts",
-    "**/packages/jest-expect/__typetests__/*.test.ts",
-    "**/packages/jest-mock/__typetests__/*.test.ts",
-    "**/packages/jest-runner/__typetests__/*.test.ts",
-    "**/packages/jest-snapshot/__typetests__/*.test.ts",
-    "**/packages/jest-types/__typetests__/config.test.ts",
-    "**/packages/jest-types/__typetests__/jest.test.ts",
-    "**/packages/jest-worker/__typetests__/*.test.ts"
+    "packages/expect/__typetests__/*.test.ts",
```

Comment:
This change improves performance. See https://github.com/jestjs/jest/pull/14969/files#r1527153647